api/base/middleware.py

```diff
@@ -156,3 +159,31 @@ def process_response(self, request, response):
                 httponly=settings.CSRF_COOKIE_HTTPONLY,
             )
         return response
+
+
+class WaffleMiddleware(MiddlewareMixin):
+    """
+    This must be overriden for our domain rules and to ensure the cookies never expire,
+    """
+    def process_response(self, request, response):
+        secure = get_setting('SECURE')
+
+        if request.environ['SERVER_NAME'] == 'localhost':
+            domain = 'localhost'
+        else:
+            domain = '.' + request.environ['SERVER_NAME']
+
+        if hasattr(request, 'waffles'):
+            for k in request.waffles:
+                name = smart_str(get_setting('COOKIE') % k)
+                active, _ = request.waffles[k]
+                response.set_cookie(
```

Comment:
This call will need to also set `SameSite='None'` on the cookie, but that doesn't appear to be supported.

Reply:
I found a way, but it ain't pretty.